portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,48 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.List; // import just the List interface
+import java.util.ArrayList; // import just the ArrayList class
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    List<String> quotes;
+
+  @Override
+  public void init() {
+    quotes = new ArrayList<>();
+    quotes.add("Live as if you were to die tomorrow. Learn as if you were to live forever. - Mahatma Gandhi ");
+    quotes.add("That which does not kill us makes us stronger. - Friedrich Nietzsche ");
+    quotes.add("Be who you are and say what you feel, because those who mind don’t matter and those who matter don’t mind. - Bernard M. Baruch ");
+    quotes.add("We must not allow other people’s limited perceptions to define us. - Virginia Satir ");
+  }
+
+  private String convertToJson(DataServlet dataServlet) {
+    String json = "{";
+    json += "\"quote1\": ";
+    json += "\"" + dataServlet.quotes.get(0) + "\"";
+    json += ", ";
+    json += "\"quote2\": ";
+    json += "\"" + dataServlet.quotes.get(1) + "\"";
+    json += ", ";
+    json += "\"quote3\": ";
+    json += "\"" + dataServlet.quotes.get(2) + "\"";;
+    json += ", ";
+    json += "\"quote4\": ";
+    json += "\"" + dataServlet.quotes.get(3) + "\"";;
+    json += "}";
+    return json;
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    DataServlet dataServlet = new DataServlet();
+    dataServlet.init();
```

Comment:
I see that we are creating a new object of DataServlet and calling init(). Could this be made simpler? For example, just calling a function getQuotesJson() which created adn populates the ArrayList and return the JSON?